src/module/actions/options/saveOptions.js

```diff
@@ -0,0 +1,20 @@
+export function saveOptions ({ currentItemState, changedItemState, options }) {
+  const returnedItemState = JSON.parse(JSON.stringify(changedItemState))
+
+  if (typeof options !== 'undefined' && Object.prototype.hasOwnProperty.call(returnedItemState, 'attributes')) {
+    if (Object.prototype.hasOwnProperty.call(options, 'sendFullAttributes')) {
+      options.sendFullAttributes.forEach(attr => {
+        if (Object.prototype.hasOwnProperty.call(returnedItemState.attributes, attr)) {
+          returnedItemState.attributes[attr] = currentItemState.attributes[attr]
+        }
+      })
+    }
+    if (Object.prototype.hasOwnProperty.call(options, 'sendUnchangedAttributes')) {
+      options.sendUnchangedAttributes.forEach(attr => {
+        returnedItemState.attributes[attr] = currentItemState.attributes[attr]
+      })
+    }
```

Comment:
I think this would be much more readable if we change this to a

``` js
switch (Object.values(options)) {
   case 'sendFullAttributes': ...
}
```

Reply:
This obsviously was ES98 shorthand for

```js
for (option in Object.values(options)) {
  switch(option)...
}
``